docs/DeveloperGuide.md

```diff
@@ -279,87 +279,87 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
 
 ### Use cases
 
-(For all use cases below, the **System** is the `SimplyKitchen` and the **Actor** is the `User`, unless specified otherwise)
+(For all use cases below, the **System** is the `SimplyKitchen` and the **Actor** is the `User`, unless otherwise specified)
 
 **Use case: Add a food item (UC01)**
 
 **Guarantees: The food item is added into the system.** 
 
 **MSS**
 
-1.  User inputs a food item.
-2.  System reads the input and adds the food item.  
+1.  User requests to add a food item.
+2.  SimplyKitchen displays changes to the food inventory.
     Use case ends.
 
 **Extensions**
 
-* 1a. System detects an error in the user's input.
+* 1a. SimplyKitchen detects an error in the user's input.
    
-   * 1a1. System displays an error message.
+   * 1a1. SimplyKitchen displays an error message and prompts the user to enter a correct input.
      
-   * 1a2. User re-enters the food item to be added again.
+   * 1a2. User enters a new food item.
       
-   * 1a3. System checks the user’s new input.
+   * 1a3. SimplyKitchen checks the user’s new input.
```

Comment:
Maybe can remove this line as the repeat step will be doing the check?

Reply:
Yes this step is redundant